.env.test

```diff
@@ -9,3 +9,7 @@ DATABASE_URL=postgres://postgres@localhost:5432/roda_test
 NOTIFY_KEY=abc
 NOTIFY_WELCOME_EMAIL_TEMPLATE=123
 REDIS_URL=redis://localhost:6379
+SECRET_KEY_BASE=super secret
```

Comment:
These are now passed in through the [existing command in travis.yml that starts the test container](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/develop/.travis.yml#L30).